scripts/common/yaml.sh

```diff
@@ -35,25 +35,25 @@ function insert_patches_strategic_merge() {
     semverParse "$kubeletVersion"
     local kubeletMinor="$minor"
 
-    # Kubernetes 1.27 uses kustomize v5 which dropped support for old, legacy style patches
-    # See: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md#changelog-since-v1270
-    if [ "$kubeletMinor" -ge "27" ]; then
-        if [[ $kustomization_file =~ "prometheus" ]] || [[ $kustomization_file =~ "rook" ]]; then
-            # TODO: multi-doc patches is not currently supported in kustomize v5
-            # continue using the deprecated 'patchesStrategicMerge' field until this is fixed
-            # Ref: https://github.com/kubernetes-sigs/kustomize/issues/5040
-            if ! grep -q "patchesStrategicMerge" "$kustomization_file"; then
-                echo "patchesStrategicMerge:" >> "$kustomization_file"
-            fi
-            sed -i "/patchesStrategicMerge.*/a - $patch_file" "$kustomization_file"
-        else
-            if ! grep -q "^patches:" "$kustomization_file"; then
-                echo "patches:" >> "$kustomization_file"
-            fi
-            sed -i "/patches:/a - path: $patch_file" "$kustomization_file"
-        fi
-        return
-    fi
+#    # Kubernetes 1.27 uses kustomize v5 which dropped support for old, legacy style patches
```

Comment:
Any reason not to remove this commented out code?

Reply:
yeah I should remove it
honestly I want to do more testing with this before I actually merge it